API/Data/ApiDbContext.cs

```diff
@@ -9,9 +9,17 @@ public class ApiDbContext : IdentityDbContext<AppUser>
         public ApiDbContext(DbContextOptions<ApiDbContext> options) : base(options)
         {
         }
+        public DbSet<Match> Matches => Set<Match>();
```

Comment:
For better clarity, we should always leave one blank line between methods and other things; tldr: add one blank line above